README.md

```diff
@@ -21,6 +21,7 @@
   - PNG (through [libspng](https://libspng.org/))
   - PPM
   - WebP (through [libwebp](https://developers.google.com/speed/webp))
+  - [StumpyCore Canvas](https://github.com/stumpycr/stumpy_core) Conversion
```

Comment:
Worth introducing a new section? Something like "integrations" and then we might put `stumpy.cr` into `src/integrations/stumpy.cr` .

Reply:
- Formats
- Integrations
- Operations

Reply:
Or maybe something better that would indicate that some things require linking to a C library and others don't.

Reply:
### Formats

#### C bindings

- JPEG (through [libjpeg-turbo](https://github.com/libjpeg-turbo/libjpeg-turbo))
- PNG (through [libspng](https://libspng.org/))
- WebP (through [libwebp](https://developers.google.com/speed/webp))

#### Native

- PPM
- [StumpyCore](https://github.com/stumpycr/stumpy_core)

### Operations

- Bilinear resize
- Box blur
- Brightness
- Channel swap
- Contrast
- Crop
- Gaussian blur
- Horizontal blur
- Vertical blur